### ISSUE TITLE
Documentation and model train details 

### ISSUE THREAD

**Issue by k-nayak** [Glutamat42/Ultra-Fast-Lane-Detection#2]
Hello @Glutamat42, 

hope you are doing great. 

I would like to know if you have provided any details about (Model + Testergebnisse Datensatz 13) such as the hyperparameters and the train details, any documentation regarding the accuracy and losses (tensorboard log may be) of the model. I couldn't find any documentation regarding that. 

Have I missed something, please do let me know. 

Thanks.

**Comment by SoraMakes** [comment]
Hello
Thanks for your interest :)

Model:
The model is exactly the same as from the original project. As far as i remember it was a small resnet with a custom head (and i think its using the output of multiple resnet blocks, not only the last one. but it may also be that this was only for training).

Hyperparams:
As far as i remember the configs (configs/d13_100%.py and configs/simple13) contain all used hyperparams. I don't think i used additional ones. So it should work if you use eg `python ufld.py configs/simple13.py --mode train`. Obviously you still would have to change paths.

I added the logged config and tensorflow files to the provided onedrive results folders (https://glutamat42.github.io/Ultra-Fast-Lane-Detection/)

Performance:
The performance according to the tusimple metric is provided as excel file in our onedrive share (https://glutamat42.github.io/Ultra-Fast-Lane-Detection/  -> "Ergebnisse"). Column C is the tusimple metric. D and E are FP/FN (also from tusimple metric). Both excel files contain the results for d13 and tusimple for every epoch of training

**Comment by k-nayak** [comment]
Hello,

1. Do you have any report or documentation as I found your part 1 (collecting and creating data from Carla and training the model very helpful and would like to recreate it)? 

2. I am also unable to load the tensorboard event file to visualize the training graphs. Any idea why? 
 I am trying this method:  (%tensorboard --logdir=path)  

Thanks

**Comment by SoraMakes** [comment]
1. I don't have a report, but you may find the howtos in our doc useful: https://glutamat42.github.io/Ultra-Fast-Lane-Detection/ld_docs/howto/howtos.html
3. Nope, works fine for me. I have the tfrecords file here: `...../tf_test/d13/events.out.tfevents.1615047940.m-gpu-instance2.84159.0` and ran tensorboard with `tensorboard --logdir=...../tf_test/`
